lib/collections/schemas/products.js

```diff
@@ -138,6 +138,48 @@ export const ProductVariant = new SimpleSchema({
       }
     }
   },
+  length: {
```

Comment:
Dimensions should not be required.

![basic_reaction_product](https://cloud.githubusercontent.com/assets/72819/23344163/7a22b19c-fcb2-11e6-8db7-7f39a8b9e9be.png)